addons/project/models/project_task.py

```diff
@@ -1338,6 +1336,28 @@ def _task_message_auto_subscribe_notify(self, users_per_task):
                     model_description=task_model_description,
                     mail_auto_delete=False,
                 )
+            unassigned_user_name = ', '.join(unassigned_users.mapped('name'))
```

Comment:
Is there a reason to send a single message with all unassignements when we send one message per person being assigned?
It would be more consistent and solve some language issues (plural vs singular, gendered vs neutral)

Reply:
Because we are sending email to person who subscribe that notification not person who is unassigned from task that's why we are sending one email with all unassignees in single email rather then sending multiple email to same follower of same task
I think we need to update message to work in for both :thinking: